effects/zio/src/test/scala/sttp/client3/impl/zio/ZioTestBase.scala

```diff
@@ -1,44 +1,10 @@
 package sttp.client3.impl.zio
 
-import sttp.client3.testing.ConvertToFuture
-import zio.{Exit, Runtime, Task, Unsafe, durationInt}
+import zio.{Task, durationInt}
 
-import scala.concurrent.{Future, Promise}
-import scala.util.{Failure, Success}
-
-trait ZioTestBase {
-  private val runtime: Runtime[Any] = Runtime.default
-
-  val convertZioTaskToFuture: ConvertToFuture[Task] = new ConvertToFuture[Task] {
-    override def toFuture[T](value: Task[T]): Future[T] = {
-      val p = Promise[T]()
-
-      unsafeRunSync(value) match {
-        case Exit.Failure(c) =>
-          p.complete(
-            Failure(
-              c.failures.headOption.orElse(c.defects.headOption).getOrElse(new RuntimeException(s"Unknown cause: $c"))
-            )
-          )
-        case Exit.Success(v) => p.complete(Success(v))
-      }
-
-      p.future
-    }
-  }
+trait ZioTestBase extends ZioRuntimeUtils {
```

Comment:
What is exactly the point in spiting this class into two ?

Reply:
The method `timeoutToNone` has a collision. It is implemented in `AbstractFetchHttpTest`, which is extended by `FetchZioHttpTest`. Because the Scala-JS code for this module did not exists yet, it did not occur. 

I think this split caused some confusion. I have reverted it and added an override for `timeoutToNone` in the FetchZio specs